packages/blocks/src/paragraph-block/paragraph-block.ts

```diff
@@ -231,6 +231,16 @@ export class ParagraphBlockComponent extends BlockElement<ParagraphBlockModel> {
       this.requestUpdate();
     });
     this.model.childrenUpdated.on(() => this.requestUpdate());
+
+    this.page.awarenessStore.slots.update.on(v => {
+      const remoteSelections = this.std.selection.remoteSelections.get(v.id);
+      const textSelection = remoteSelections?.find(
+        selection => selection.type === 'text'
+      );
+      if (textSelection) {
```

Comment:
Why we need to check `textSelection` here?

Reply:
Suppose there are two users A and B. User A creates `#`. At this time, user B only needs to update the placehoder when it senses data changes. Here, `remoteSelections` is mainly used to determine whether it is user B.

Only when it is a text selection, it means that the user is in editing mode. This can avoid unnecessary updates.